integration/lql_create_test.go

```diff
@@ -38,8 +38,10 @@ func TestQueryCreateHelp(t *testing.T) {
 
 func TestQueryCreateEditor(t *testing.T) {
 	out, err, exitcode := LaceworkCLIWithTOMLConfig("query", "create")
-	assert.Contains(t, out.String(), "Type a query to create")
-	assert.Contains(t, out.String(), "[Enter to launch editor]")
+	assert.Contains(t, out.String(), "Choose query language to create")
```

Comment:
I would not expect this type of prompt to be released until Lacework officially supports Rego for all customers.

Can we make the argument an optional string that we don't prompt the customer for as long as we are in private preview? This way customers that are not part of the preview are not impacted by Rego support and customers that are part of the preview can use the appropriate flag.

Reply:
I've added a FF check to ensure the prompt to choose a query language is only displayed to customers for whom Rego is enabled.